src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -296,5 +296,141 @@ def spinningBody(show_plots, cmdTorque, lock, thetaRef):
     return [testFailCount, ''.join(testMessages)]
 
 
+@pytest.mark.parametrize(
+    "max_lim, min_lim", [(-70, 70), (-10, -20)]  # cases: max less than min, start angle inside bounds
+)
+def test_spinning_body_enforces_limits(max_lim, min_lim):
+    """Verify that model raises exception when invalid limits are specified.
+    
+    :param float max_lim: Maximum angle limit [deg]
+    :param float min_lim: Minimum angle limit [deg]
+    """
+    spinningBody = spinningBodyOneDOFStateEffector.SpinningBodyOneDOFStateEffector()
+    spinningBody.sHat_S = [0, -1, 0]
+    spinningBody.theta_max = np.deg2rad(max_lim)
+    spinningBody.theta_min = np.deg2rad(min_lim)
+
+    with pytest.raises(ValueError):
+        spinningBody.Reset(0)
```

Comment:
This test doesn't have an assert statement?  It passes regardless of what values I put in it?

Reply:
Squash this with the prior commits to clean up the commit history.